readme.md

```diff
@@ -23,3 +23,262 @@ For the revision task, please refer to: [revision.md](revision.md).
+- **Bitwise operator\:**
+  - **Bitwise AND(`&`)\:** used to perform a bitwise AND operation between the corresponding bits of two integer operands by comparing the bits at each position. It returns an integer result where each bit position contains the result of the OR operation on the corresponding bits of the operands.
+ ![alt text](image-10.png)
+   - **Bitwise OR(`|`)\:** used to perform a bitwise OR operation between the corresponding bits of two integer operands by comparing the bits at each position. It returns an integer result where each bit position contains the result of the OR operation on the corresponding bits of the operands.
+  ![alt text](image-9.png)
+  - **Bitwise XOR(`^`)\:** return 1 if the corresponding bits of two operands are opposite.
+  ![alt text](image-11.png)
+  - **Bitwise complement\:** a unary operator works on only one operand by changing 1 to 0 and 0 to 1.
+  ![alt text](image-12.png)
+  - **Shift operator\:**
+    - Right shift operator (`>>`): shifts all bits towards right by certain number of specified bits.
+    ![alt text](image-13.png)
+    - Left shift operator(`<<`): shifts all bits towards left by certain number of specified bits.
+    ![alt text](image-14.png)
+- **Difference between logical AND and bitwise AND\:**
+
+| Aspect           | Bitwise AND (`&`)                                            | Logical AND (`&&`)                                          |
+| ---------------- | ------------------------------------------------------------ | ----------------------------------------------------------- |
+| Operand Type     | Operates on integer operands (integral types).               | Operates on boolean operands (`int`, `char`, `bool`, etc.). |
+| Usage            | Performs bitwise AND operation between corresponding bits.   | Performs logical AND operation between boolean expressions. |
+| Result           | Returns an integer with bits set based on the AND operation. | Returns a boolean (`true` or `false`) based on logical AND. |
+| Short-circuiting | Does not perform short-circuit evaluation.                   | Performs short-circuit evaluation.                          |
+| Evaluation       | Both operands are always evaluated.                          | Right operand is only evaluated if left operand is `true`.  |
+| Example          | `int result = num1 & num2;`                                  | `if (condition1 && condition2) { /* code */ }`              |
+
+=> We can not interchange `&` and `&&` for each other. Otherwise, it could lead to serious error.
+# 6. ASCII
+- **Definition\:** ASCII stands for American Standard Code for Information Interchange. It's a character encoding standard that assigns numerical values to characters. ASCII codes represent text in computers, communications equipment, and other devices that use text.
+  <br>Here are some examples of characters in ASCII table:
+
+| ASCII (Decimal) | ASCII (Hexadecimal) | Character |
```

Comment:
@spm1102 
`#reply_required`
`#fix_required`

![image](https://github.com/EDABK-C-C-Training-K67/week2-cfundamental-themostbasicdefinition-spm1102/assets/94220196/05aedb1d-eb1c-4325-8bbc-de445e6285da)

Provide the table list **full ASCII code** (0 - 127)

Reply:
@Dark-GreenCat oh it's my mistake. I've already add more.